Formula/pcapplusplus.rb

```diff
@@ -0,0 +1,36 @@
+class Pcapplusplus < Formula
+  desc "Multiplatform C++ network sniffing and packet parsing and crafting"
+  homepage "https://seladb.github.io/PcapPlusPlus-Doc"
+  url "https://github.com/seladb/PcapPlusPlus/archive/v18.08.tar.gz"
+  sha256 "dff6f7c677b2050f880043b125e984238cd8af0f1c25864e09e87fb8d71ec9ab"
+
+  def install
+    system "./configure-mac_os_x.sh", "--install-dir", prefix
+
+    # library requires to run 'make all' and 'make install' in two separate commands
```

Comment:
Can we nudge this across two lines. Almost everything should be wrapped to 80 characters. Everything ideally, but obviously not always practical; comments are easy to wrap to a character limit though.

Reply:
Done